Sources/Options.swift

```diff
@@ -41,7 +41,7 @@ public struct Options {
     }
 
     private static func parseMediaUrl(mediaUrl: String) throws -> VideoInfo {
-        let regex = "https:/.*[/](vod|live)([/]|[/.][^/]*[/])([^/^.]*)[/.].*"
+        let regex = "https:/.*[/](?<type>vod|live).*/(?<id>(vi|li)[^/^.]*)[/.].*"
```

Comment:
Named group `<type>` and `<id>` are defined, you should use them:
Example: https://whatdidilearn.info/2018/07/29/how-to-capture-regex-group-values-in-swift.html